images/scripts/centos-8-stream.install

```diff
@@ -5,4 +5,7 @@ set -e
 # remove cockpit distro packages, testing with upstream master
 rpm --erase --verbose cockpit cockpit-ws cockpit-bridge cockpit-system
 
+# HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1879851
+rm -f /var/lib/sss/db/*
```

Comment:
Shouldn't this be in *.setup?

Reply:
I prefer doing such hacks in *.install. It's much easier to experiment with/change them without having to do an image rebuild, and it would need an extra condition in the shared rhel.setup.

Reply:
Ok, fair enough.